<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences_window</class>
 <widget class="QMainWindow" name="preferences_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/icons/gears.gif</normaloff>:/icons/icons/gears.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="e_default_open_dir"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="b_default_open_dir">
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/folder.gif</normaloff>:/icons/icons/folder.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Default Open Directory:</string>
      </property>
      <property name="buddy">
       <cstring>e_default_open_dir</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="e_default_save_dir"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Default Save Directory:</string>
      </property>
      <property name="buddy">
       <cstring>e_default_save_dir</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>UTF-8 Encoding:</string>
      </property>
      <property name="buddy">
       <cstring>e_default_utf_8</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="e_default_utf_8"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="b_default_save_dir">
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/folder.gif</normaloff>:/icons/icons/folder.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="e_default_font_result">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Default font editor:</string>
      </property>
      <property name="buddy">
       <cstring>e_default_font_editor</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="e_default_font_editor">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="b_default_font_editor">
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/font_type.gif</normaloff>:/icons/icons/font_type.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Default font result:</string>
      </property>
      <property name="buddy">
       <cstring>e_default_font_result</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="b_save">
      <property name="text">
       <string>Save preferences</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/disk.gif</normaloff>:/icons/icons/disk.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="b_default_font_result">
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icons/icons/font_type.gif</normaloff>:/icons/icons/font_type.gif</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Result editable:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="e_default_editable">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>e_default_open_dir</tabstop>
  <tabstop>b_default_open_dir</tabstop>
  <tabstop>e_default_save_dir</tabstop>
  <tabstop>b_default_save_dir</tabstop>
  <tabstop>e_default_utf_8</tabstop>
  <tabstop>e_default_font_editor</tabstop>
  <tabstop>b_default_font_editor</tabstop>
  <tabstop>e_default_font_result</tabstop>
  <tabstop>b_default_font_result</tabstop>
  <tabstop>e_default_editable</tabstop>
  <tabstop>b_save</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b_default_open_dir</sender>
   <signal>clicked()</signal>
   <receiver>preferences_window</receiver>
   <slot>slot_b_default_open_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_default_save_dir</sender>
   <signal>clicked()</signal>
   <receiver>preferences_window</receiver>
   <slot>slot_b_default_save_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_default_font_editor</sender>
   <signal>clicked()</signal>
   <receiver>preferences_window</receiver>
   <slot>slot_b_default_font_editor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_default_font_result</sender>
   <signal>clicked()</signal>
   <receiver>preferences_window</receiver>
   <slot>slot_b_default_font_result()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_save</sender>
   <signal>clicked()</signal>
   <receiver>preferences_window</receiver>
   <slot>slot_b_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_b_default_font_editor()</slot>
  <slot>slot_b_default_font_result()</slot>
  <slot>slot_b_default_open_dir()</slot>
  <slot>slot_b_default_save_dir()</slot>
  <slot>slot_b_save()</slot>
 </slots>
</ui>
